Soccar [Client]/test/nl/soccar/library/SessionControllerTest.java

```diff
@@ -7,64 +7,80 @@
 import static org.junit.Assert.*;
 
 /**
+ * JUnit test that tests the nl.soccar.library.SessionController class.
  *
  * @author PTS34A
  */
 public class SessionControllerTest {
-    
+
     // Declaration of test objects.
-    private SessionController controller;
+    private SessionController sessionController;
     private Session session;
-    private Player player;
+    private Player player1;
     private Player player2;
-    
-    public SessionControllerTest() {
-    }
-    
+    private Player player3;
+    private Player player4;
+
+    /**
+     * Instantiation of test objects.
+     */
     @Before
     public void setUp() {
-        controller = new SessionController();
-        session = new Session("Testroom", "1234");
-        player = new Player("Testuser", "password", Privilege.NORMAL, CarType.CASUAL);
-        player2 = new Player("Testuser2", "password", Privilege.NORMAL, CarType.CASUAL);
+        sessionController = new SessionController();
+        session = new Session("name", "password");
+        player1 = new Player("username", "password", Privilege.NORMAL, CarType.CASUAL);
+        player2 = new Player("password", "password", Privilege.NORMAL, CarType.CASUAL);
+        player3 = new Player("password", "password", Privilege.NORMAL, CarType.CASUAL);
+        player4 = new Player("password", "password", Privilege.NORMAL, CarType.CASUAL);

```

Comment:
Typo gefixt. Heb alle usernames aangepast naar 'username' om de test zo abstract mogelijk te maken (ingevulde username is niet van belang).